fireant/slicer/queries/builder.py

```diff
@@ -82,6 +85,14 @@ def queries(self):
         """
         raise NotImplementedError()
 
+    def fetch(self, hint=None):
+        queries = [query.hint(hint)
+                   if hint is not None
```

Comment:
Might be worth making a utility function for this, as the same list comprehension is used in 3 different places.

Reply:
even better would be to go through the base class implementation every time. Was looking at that but decided to not do it right away.